### DISCUSSION TITLE
Hi Gerry, could you help me get my sb 500 up and running

### DISCUSSION THREAD

**Discussion by purdyson4** [GerryDazoo/Slinger#331] [General]
Thanks,
John jap64@me.com

**Comment by purdyson4** [comment]
Hi Gerry, I am really interested in trying to use my sb500 still if possible?
Thanks,
John

**Reply by DoogieHowitzerWMD** [reply]
First, did you manage to pull the admin password for the box before Sling's servers went down?

Not having that is prohibitive to getting the 500's to run with Gerry's Slinger server. Maybe someone finds a workaround for that down the road, but that's how things stand right now.

**Reply by GerryDazoo** [reply]
It may be in the slingplayer console.log file

**Reply by DoogieHowitzerWMD** [reply]
> It may be in the slingplayer console.log file

That's for the desktop version, correct? I only ever had the web player version and I couldn't find it from a locally stored file.

**Comment by purdyson4** [comment]
Hi Gerry, how can I check the console.log ***@***.*** Nov 21, 2022, at 11:40 AM, GerryDazoo ***@***.***> wrote:﻿
It may be in the slingplayer console.log file

**Comment by purdyson4** [comment]
Hi Gerry, I’m scared to even plug in my slingbox, I unplugged everything before the ***@***.*** Nov 21, 2022, at 11:40 AM, GerryDazoo ***@***.***> wrote:﻿
It may be in the slingplayer console.log file

**Comment by purdyson4** [comment]
Originally I used my iPhone to set up/watch my slingbox, I have a Mac ***@***.*** Nov 21, 2022, at 1:15 PM, DoogieHowitzerWMD ***@***.***> wrote:﻿

It may be in the slingplayer console.log file

That's for the desktop version, correct? I only ever had the web player version and I couldn't find it from a locally stored file.

**Reply by DoogieHowitzerWMD** [reply]
I'm not sure if anyone has a way to pull the password from Slingplayer app on iphone.

Did you say you also had a slingplayer program on a Mac? I think might be an easier route.

**Comment by purdyson4** [comment]
I do, but it says I need to update, which I am hesitant to do,I would be forever in your debt if you could help me get my slingbox ***@***.*** Nov 22, 2022, at 12:49 PM, DoogieHowitzerWMD ***@***.***> wrote:﻿
I'm not sure if anyone has a way to pull the password from Slingplayer app on iphone.
Did you say you also had a slingplayer program on a Mac? I think might be an easier route.

**Reply by DoogieHowitzerWMD** [reply]
No, you don't need to worry about updating the slingplayer program on the Mac. You don't need to open the program itself at all.

Per what Drano32 said in a post in another thread, first you have to go into your Mac user folder, then into the "Library" folder, then into the "Application Support" folder, and then into the "Slingplayer Desktop" folder. In the Slingplayer Desktop folder, you need to find the file called "console.log". 

That will contain the admin password. The password in the file will either be a 15-character code, or it will start with "E1:" and have 64 characters. If it is the latter, there is a user here who can decrypt it and get the 15-character password for you.
